exercises/saddle-points/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+
+pub fn find_saddle_points(input: Vec<Vec<u64>>) -> Vec<(u64, u64)>{
```

Comment:
There's no need to transfer ownership of the input for `find_saddle_points` to work. We want to teach people how to write idiomatic Rust, so in this case, this function should take `&[&[u64]]` (reference to slice of reference to slice of u64). `Vec<Vec<u64>>` should coerce transparently to the slice form.